Automattic/jetpack
# [Boost] Add admin notification bubble on cloud css failure

Fixes https://github.com/Automattic/boost-cloud/issues/160.

#### Changes proposed in this Pull Request:

* This PR adds a notification bubble next to Boost if there's a problem with Cloud CSS.

![image](https://user-images.githubusercontent.com/11799079/207003514-04b02d7c-f111-401e-9563-b7dbbbcad9af.png)

#### Other information:

For the time being, the code only checks if something's wrong with Cloud CSS. It should be easy to update it to include another Boost feature if we need to (like say the Image Guide).

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Enable "**Automatically Optimize CSS Loading**" from Boost's dashboard;
* Setup your test website to use your local Boost Cloud;
* Break the Cloud CSS generation;
* Make a request to regenerate CSS through the WP-admin;
* Once you get the error, refresh the page;
* You should see the number 1 in red (dark orange?) next to Boost in the sidebar.
